charts/kuberpult/Makefile

```diff
@@ -16,6 +16,11 @@
 #Copyright 2021 freiheit.com
 MAKEFLAGS += --no-builtin-rules
 
+# Container image as recommended by https://github.com/helm/chart-testing#installation
+#CT := podman run -ti quay.io/helmpack/chart-testing:v3.5.0 ct
+# Manual / local installation
+CT := ct
```

Comment:
This change does not reflect what is written in the PR description.
Shouldn't running ct with the docker image be the default (as it has always the correct version)? I think manually running is an exception.